src/dsp_tools/commands/xmlupload/resource_create_client.py

```diff
@@ -156,7 +156,8 @@ def _make_bitstream_file_value(bitstream_info: BitstreamInfo) -> dict[str, Any]:
         case "mp4":
             prop = "knora-api:hasMovingImageFileValue"
             value_type = "MovingImageFileValue"
-        case "jpg" | "jpeg" | "jp2" | "png" | "tif" | "tiff":
+            # jpx is the extension of the files returned by dsp-ingest
```

Comment:
I suggest moving this comment 2 lines down, because it doesn't belong to mp4, but the images.

Reply:
But then, the indentation is misleading. I would then de-indent it, so that it is aligned with the `case`